Training24/ClassicLibrary.c

```diff
@@ -1,12 +1,35 @@
 // ------------------------------------------------------------------------------------------------
 // Training ~ A training program for new joiners at Metamation, Batch - July 2024.
 // Copyright (c) Metamation India.
+// Kalaimagal V P
 // ------------------------------------------------------------------------------------------------
-// Classiclibrary.c
-// Program on main branch.
+// ClassicLibrary.c
+// Program on A5 branch.
+// Write a program that sorts integer arrays in ascending order using any of the sorting algorithms discussed

```

Comment:
Don't have sentences like the above, Just Give the simple description of what your program does